uwrt_mars_rover_blackfly_calibration/calibration_data/left_18475242.txt

```diff
@@ -0,0 +1,31 @@
+# oST version 5.0 parameters
```

Comment:
is this generated from ros?

Reply:
yep by the cameracalibrator node. ig its meant to be a human readable version of the yaml file, but the yaml file is already pretty readable. i think I should delete the txt file. what do u think?

Reply:
probs fine to leave it since the next person to run cameracalibrator will get thgis output too